libcore/BookmarkList.vala

```diff
@@ -177,14 +177,15 @@ namespace Files {
          * only permits one bookmark per uri.
          */
         public void rename_item_with_uri (string uri, string new_name) {
-            foreach (unowned Files.Bookmark bookmark in list) {
+            foreach (var bookmark in list) {
```

Comment:
there is no real reason to use a full reference here though

Reply:
There is no significant cost though is there?  I did this just to be on the safe side ...

Reply:
Reverted to unowned var